File: SampleConsoleApp/TranslateResponseModel.cs
namespace ACW_GenAI_UniversalTranslator
{
    public class TranslateResponseModel
    {
        public DetectedLanguage? detectedLanguage { get; set; }
        public Translations[]? translations { get; set; }
    }

    public class DetectedLanguage
    {
        public string? language { get; set; }
        public double? score { get; set; }
    }

    public class Translations
    {
        public string? text { get; set; }
        public string? to { get; set; }
    }
}

File: SampleConsoleAppJava/TranslateResponseModel.java
public class TranslateResponseModel {
    private DetectedLanguage detectedLanguage;
    private Translations[] translations;

    public DetectedLanguage getDetectedLanguage() {
        return detectedLanguage;
    }

    public void setDetectedLanguage(DetectedLanguage detectedLanguage) {
        this.detectedLanguage = detectedLanguage;
    }

    public Translations[] getTranslations() {
        return translations;
    }

    public void setTranslations(Translations[] translations) {
        this.translations = translations;
    }

    public static class DetectedLanguage {
        private String language;
        private double score;

        public String getLanguage() {
            return language;
        }

        public void setLanguage(String language) {
            this.language = language;
        }

        public double getScore() {
            return score;
        }

        public void setScore(double score) {
            this.score = score;
        }
    }

    public static class Translations {
        private String text;
        private String to;

        public String getText() {
            return text;
        }

        public void setText(String text) {
            this.text = text;
        }

        public String getTo() {
            return to;
        }

        public void setTo(String to) {
            this.to = to;
        }
    }
}
